panel/panel.py

```diff
@@ -207,107 +177,115 @@ def fill_panel(window, update=False, **panel_filter):
     else:
         settings.set("sublime_linter_panel_filter", panel_filter)
 
-    to_render = RenderLines()
-    for vid, view_dict in errors.items():
-        if util.is_none_or_zero(view_dict["we_count_view"]):
-            continue
-
-        view_lines = get_view_lines(view_dict, panel_filter)
-        if view_lines:
+    to_render = []
+    for bid, buf_dict in errors.items():
```

Comment:
`buf_dict` is actually a list these are the `errors`, no?

what you call `errors`above is probably `all_errors` or `errors_by_bid`

Reply:
Just to ensure I understand you here: your comment is in regards of variable names not logic?

Reply:
Yes